### PR TITLE
Fix: Quad Cannon Missing Recoil And Muzzle Flash When Firing At Grounded Targets

### PR DESCRIPTION
**Pull request by commy2**
- partial fi.x for #1332

### CODE REVIEW

File: Patch104pZH/GameFilesEdited/Data/INI/Object/GC_Slth_GLAUnits.ini [hunk 1]
```diff
@@ -7047,12 +7048,28 @@ Object GC_Slth_GLAVehicleQuadCannon
...
       WeaponMuzzleFlash = SECONDARY BarrelFX
       WeaponRecoilBone  = SECONDARY Barrel
-      WeaponFireFXBone  = TERTIARY   BarrelMS
-      WeaponMuzzleFlash = TERTIARY   BarrelFX
-      WeaponRecoilBone  = TERTIARY   Barrel
+      ShowSubObject = TURRET
+      HideSubObject = TURRETUP01 TURRETUP02 BarrelFX01 BarrelFX02 BarrelFX03 BarrelFX04  ; Since we are showing one of three turrets, we need to then hide the subobject muzzleflash
+    End
+
+    ConditionState = USING_WEAPON_C
```
Comment (xezon) [inline Patch104pZH/GameFilesEdited/Data/INI/Object/GC_Slth_GLAUnits.ini:7066]: Why do some of the Quads have 3 weapons?

Comment (commy2) [reply Patch104pZH/GameFilesEdited/Data/INI/Object/GC_Slth_GLAUnits.ini:7066]: The GLA02 mission Stealth General Quad Cannon has a "Quad Cannon Sniper" upgrade, that gives it essentially a USA Pathfinder firing mode when targeting infantry (one shots).

**Review by xezon — APPROVED** [review]
Looks very good.

**Comment by commy2** [comment]
https://user-images.githubusercontent.com/6576312/194764236-2064b105-d900-46f2-b27f-2cd6951f70c4.mp4

**Pull request merged**